edk2toolext/invocables/edk2_multipkg_aware_invocable.py

```diff
@@ -20,6 +20,19 @@ class MultiPkgAwareSettingsInterface():
         and which functions can be implemented.
      '''
 
+    SettingsClass = None  # this will be filled in by the edk invocable
```

Comment:
Why cache if we already have a method? Is it important to work around the MRO?

Reply:
No, I just put it in there so it will be populated by autocomplete and it shows up as a class member

Reply:
Since we don't have access to the GetSettingsClass method in the SettingsManager, this is here to cache it inside the settings